### ISSUE TITLE
Training details on single dataset(3DPW、EMDB and COCO)

### ISSUE THREAD

**Issue by zhanganny** [g-fiche/VQ-HPS#3]
Hello~ Sorry to bother you again. When I train on 3DPW with all the other config in the repo not changed, I find the loss of validation increase. It seems abnormal. What's the training detail of your results on single dataset showed in the paper, such as training epoch and other config detail? How can I reproduce it? Thank you so much!

<img width="460" alt="image" src="https://github.com/user-attachments/assets/e87fc036-f582-4fb3-ba1d-cdd1e7117030">

**Comment by g-fiche** [comment]
Hello,
No problem! We observed the same thing. This is due to the cross-entropy loss, which increases on the validation set.

We investigated this "issue", and we believe that this is because some embeddings in the dictionary of the Mesh-VQ-VAE are very similar. Then, the model sometimes predicts the wrong index but a rather good embedding that, once decoded, gives good 3D performance.

Even if the validation loss increases, the validation 3D metrics should decrease.

**Comment by ouusan** [comment]
@zhanganny Hi, bro! It looks like you’ve successfully run this code. Have you encountered this issue before? How did you solve it ?
I try to install the code with only 3DPW dataset. In this line: l1_loss(pred_2d, gt_2d), the shape of pred_2d is [32,52,2] while shape of gt_2d is [32,24,2], why this occures and what should i do to solve this ? Thankyou~ **my error details are there: https://github.com/g-fiche/VQ-HPS/issues/6**
